modules/migrating-to-multi-arch-cli.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * updating/migrating-to-multi-payload.adoc
+
+:_content-type: PROCEDURE
+[id="migrating-to-multi-arch-cli_{context}"]
+= Migrating clusters to a multi-arch/heterogenous payload using the CLI
+
+.Prerequisites
+
+* You must be logged in as a user with `cluster-admin` privileges on all clusters.
+
+.Procedure
+
+. Install your {product-title} cluster on your preferred platform. For more information on selecting a supported platform for your cluster installation, see _Selecting a cluster installation type_ in the Additional resources section.
+. Set the channel to candidate-4.13 with `oc adm upgrade channel candidate-4.13`. For more information about channels, refer to _Understanding update channels and releases_ listed in the Additional resources section.
++
+[source,terminal]
+----
+$ oc adm upgrade channel candidate-4.13
+----
+
+. Upgrade to multi-arch with the `oc adm upgrade --to-multi-arch` command:
++
+[source,terminal]
+----
+oc adm upgrade --to-multi-arch.
+----
+
```

Comment:
I wonder whether we should set some expectations about hwat will happen (people like their actions to have an observable effect). I don't think there's an observable effect of this action - the cluster won't spend any noticeable time migrating / upgrading afaik.

Reply:
@evakhoni @shellyyang1989 WDTY? :)

Reply:
Yeah, I like Petr's idea! We might need to add an example output to show the cluster has been migrated to multi-arch between the same version, and/or a step to show how to check the payload arch.

BTW, the command has a period at the end.

> $ oc adm upgrade --to-multi-arch.

$ oc adm upgrade --to-multi-arch

Reply:
We also want to suggest the users run some commands like:

```
watch 'oc get co && oc get mcp && oc get clusterversion'
```

to follow the complete process.

cc @Prashanth684

Reply:
agreed. we need to make users aware that the cluster operators will undergo an "upgrade" and would need to be monitored with the above commands mentioned. ref: https://github.com/openshift/openshift-docs/pull/56591

Reply:
`watch 'oc get co; oc get mcp; oc get clusterversion` will be confusing for customers. We should just provide one command which they can use to watch if the upgrade is done. Also tracking a multi-arch upgrade is not different than the usual upgrade. So we should not suggest something special here.